src/main/java/org/example/homework3/Task2.java

```diff
@@ -0,0 +1,27 @@
+package org.example.homework3;
+
+public class Task2 {
+
+    public static void main(String[] args) {
+
+        int a = 5;
+        int b = 10;
+
+        if (a>b) {
```

Comment:
```suggestion
        if (a > b) {
```
Use "Format Code" IDEA feature